examples/esp32-wroom-32/slipdev/main.c

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright (C) 2022 Mesh4all <contact@mesh4all.org>
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ */
+
+/**
+ * @ingroup     examples/esp32-wroom
```

Comment:
```suggestion
 * @ingroup     examples_esp32-wroom
```
Why not the current form?